.eslintrc.json

```diff
@@ -83,6 +83,13 @@
         "ecmaVersion": 6
       }
     },
+    {
+      /* This module uses ES8 async / await due to WebXR Anchor Module integration */
```

Comment:
Comment incorrectly refers to WebXR Anchor Module. That being said, I don't believe `async/await` is a hard requirement for `splat.js` and it can be rewritten in the A-Frame code style for consistency, avoiding this exception altogether.